android/capacitor/src/main/java/com/getcapacitor/Bridge.java

```diff
@@ -397,6 +398,22 @@ public boolean launchIntent(Uri url) {
         return false;
     }
 
+    public boolean onRenderProcessGone(final WebView view, RenderProcessGoneDetail detail) {
+        /*
+         * Give plugins the chance to handle or record render process removal
+         */
+        boolean result = false;
+        for (Map.Entry<String, PluginHandle> entry : plugins.entrySet()) {
+            Plugin plugin = entry.getValue().getInstance();
+            if (plugin != null) {
+                if (plugin.onRenderProcessGone(view, detail)) {
+                    result = true;
```

Comment:
The loop continues irrespective of the return value so that any reporting plugins can find out about the crash even if one of the plugins is attempting to handle it gracefully.